sdk/test-utils/recorder/test/recorder.spec.ts

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { expect } from "chai";
```

Comment:
Can we use `assert` from `vitest` instead such as the following?  Also, `vitest` exposes `expect` so we don't need it from `chai`.

```
// From
    expect(context).to.eql({
      suiteTitle: "vitest suite title",
      testTitle: "vitest test title",
    });

// To
assert.equal(context, { 
  suiteTitle: "vitest suite title",
  testTitle: "vitest test title",
});
```

Reply:
updated